counter/Classes/Presentation/Scenes/Main/ViewControllerReactor.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ViewControllerReactor.swift
+//  counter
+//
+//  Created by Pich on 2/27/19.
+//  Copyright © 2019 pich. All rights reserved.
+//
+
+import ReactorKit
+import RxSwift
+
+class ViewControllerReactor: Reactor {
+    enum Action {
+        case addBtnClicked
+        case minusBtnClicked
+    }
+    enum Mutation {
+        case add
+        case minus
+    }
+    struct State {
+        var counter: Int
+    }
+    var initialState: ViewControllerReactor.State
```

Comment:
normally we initialize State in this line 
`
var initialState: ViewControllerReactor.State = State(
     counter: 0
)
`

Reply:
yeah yesterday you told me, I will correct it